cmd/synchronization/app/options.go

```diff
@@ -43,7 +45,16 @@ func (p *Providers) UnmarshalJSON(data []byte) error {
 	}
 	depURL, err := url.Parse(rawProviders.Deployinator)
 	p.DeployinatorURL = depURL
-	return errors.Wrap(err, "unable to parse Deployinator URL")
+	if err != nil {
+		return errors.Wrap(err, "unable to parse Deployinator URL")
+	}
+
+	ogUrl, err := url.Parse(rawProviders.OpsgenieIntegrationsManager)
+	p.OpsgenieIntegrationsManagerURL = ogUrl
```

Comment:
Move below after error check

Reply:
I copied the lines above it, would you like me to change those within this PR?